default.xml

```diff
@@ -63,7 +63,9 @@
     <linkfile src="root.bp" dest="Android.bp" />
     <linkfile src="bootstrap.bash" dest="bootstrap.bash" />
   </project>
+<!-- removed because it's java stuff
   <project path="art" name="android_art" groups="pdk" remote="halium" />
```

Comment:
Removing art completely also means removing libsigchain which is being used by (at least) libsurfaceflinger it seems. Seems that building of libsurfaceflinger was disabled in 7.1 as well, so I guess we're good to disable it in 8.1 as well. I'll patch that in the android_frameworks_native similar to what was done for 7.1: https://github.com/Halium/android_frameworks_native/commit/9a547f8e95da7c866b479d9e776d2bae3d2cf78c#diff-d753eba7f87fba97856e0ca871ed5000